Sources/Player/Player.docc/Articles/metrics/metrics-article.md

```diff
@@ -0,0 +1,46 @@
+# Metrics
+
+@Metadata {
+    @PageColor(purple)
+    @PageImage(purpose: card, source: metrics-card, alt: "An image depicting a waveform inspected by a looking glass.")
+}
+
+Inspect key metrics during playback.
+
+## Overview
+
+Providing the best playback experience to your users is crucial. This can prove to be challenging, especially since media playback involves many moving parts (stream encoding and packaging, metadata delivery, CDN, network quality), all possibly negatively impacting the end user experience in various ways.
+
+> Tip: More information about stream encoding and packaging please refer to <doc:stream-encoding-and-packaging-advice-article>.
+
+To better understand how your playback experience is perceived by your users, you usually need to ask yourself:
+
+- How long does a user wait for content to be ready to play?
+- How many playback sessions fail with an error, and which errors occur the most?
+- Do stalls occur often and for which reasons? How much users do have to wait for playback to resume?
+
+Pillarbox ``Player`` provides extensive metrics which can help you provide answer to these questions.
+
+## Obtain metrics periodically
+
+Subscribe to ``Player/periodicMetricsPublisher(forInterval:queue:limit:)`` to receive a stream of ``Metrics`` related to the item currently being played.
+
+These metrics contain various instantaneous measurements, e.g. a bandwidth estimate or the bitrate of the content being played, but also cumulative values and increments since the last measurement. They are especially useful to understand not only what happens globally during a playback session, but also at regular time intervals.
+
+## Receive important metric-related events
+
+Subscribe to ``Player/metricEventsPublisher`` to receive a stream of ``MetricEvent``s related to the item currently being played.
+
+Events include timing events, most notably metadata and asset loading, as well as fatal and non-fatal errors. Please refer to the documentation to learn more about all ``MetricEvent/Kind-swift.enum``s of events that are currently supported.
```

Comment:
```suggestion
Events include timing events, most notably metadata and asset loading, as well as warnings and errors. Please refer to the documentation to learn more about all ``MetricEvent/Kind-swift.enum``s of events that are currently supported.
```

In the code, we are using the term **warning** and not **non-fatal**. Isn't it easier to keep using these terms?

Reply:
Good point